src/benchmark_environments/classic_control.py

```diff
@@ -19,6 +19,19 @@ class FixedHorizonCartPole(gym.envs.classic_control.CartPoleEnv):
     in `TimeLimit` with max steps 500.)
     """
 
+    def __init__(self):
+        """Builds FixedHorizonCartPole, modifying observation_space from Gym parent."""
+        super().__init__()
+
+        high = [
+            np.finfo(np.float32).max,  # x axis
+            np.finfo(np.float32).max,  # x velocity
+            np.pi,  # theta in radians
```

Comment:
Ah, I wonder if the incorrect obs space contributed to my trouble with training this env on PPO before moving shifting the observations to [-pi, pi] from [0, 2*pi].

Reply:
I don't think PPO directly uses the observation space so probably unrelated.